int32 shape   # 1: rec, 2: cir

float64 latitude
float64 longitude

float64 width
float64 height

float64 radius

---
bool result